doc/source/whatsnew/v0.23.4.txt

```diff
@@ -32,7 +32,7 @@ Bug Fixes
 
 - Bug where calling :func:`DataFrameGroupBy.agg` with a list of functions including ``ohlc`` as the non-initial element would raise a ``ValueError`` (:issue:`21716`)
 - Bug in ``roll_quantile`` caused a memory leak when calling ``.rolling(...).quantile(q)`` with ``q`` in (0,1) (:issue:`21965`)
--
+- Bug in ``DataFrame.groupby`` caused ValueError in Py3 when applying stats function over column level with skipna=False (:issue `19720`)
```

Comment:
Use the `func:` type reference for `DataFrame.GroupBy` and put double backticks around literals like `ValueError`.

I'd also remove the word stats and change "column level" to "axis=1". Does this affect more methods besides sum?

Reply:
Yes, it also affects the methods mean, median, min, max and possibly others.